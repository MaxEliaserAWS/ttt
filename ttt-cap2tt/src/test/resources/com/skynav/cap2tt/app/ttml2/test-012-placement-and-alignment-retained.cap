Lambda字幕V4	DF0+1	SCENE"和文標準"

1	00000300/00000600	vertical	＠縦右！ ＠中頭！
2	00000700/00001000	vertical
3	00001100/00001400	vertical
