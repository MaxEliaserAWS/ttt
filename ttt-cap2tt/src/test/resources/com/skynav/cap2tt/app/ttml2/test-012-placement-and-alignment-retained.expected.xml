<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttva:model="ttml2" xml:lang="ja" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" tts:textAlign="center"/><style xml:id="s2" tts:textAlign="start"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/><region xml:id="縦右" tts:extent="15rw 80rh" tts:position="right 10rw center" tts:writingMode="tbrl"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:00:03:00" end="00:00:06:00" region="縦右" style="s1"><span style="s2">vertical</span></p><p begin="00:00:07:00" end="00:00:10:00" region="縦右" style="s1"><span style="s2">vertical</span></p><p begin="00:00:11:00" end="00:00:14:00" region="縦右" style="s1"><span style="s2">vertical</span></p></div></body></tt>
